### PR TITLE
108314 Added endpoint to sort and filter on documents

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: zgw/documenten-api/src/test/kotlin/com/ritense/documentenapi/client/DocumentenApiClientTest.kt
Comment (theo-ritense) [inline zgw/documenten-api/src/test/kotlin/com/ritense/documentenapi/client/DocumentenApiClientTest.kt:1]: I'm missing a test for the page size exception

**Review by theo-ritense — DISMISSED** [review]
Looks good, aside from the one test you could add.

File: zgw/documenten-api/src/test/kotlin/com/ritense/documentenapi/client/DocumentenApiClientTest.kt
Comment (ivo-ritense) [reply zgw/documenten-api/src/test/kotlin/com/ritense/documentenapi/client/DocumentenApiClientTest.kt:1]: Added a test for this scenario

**Comment by marijnritense** [comment]
Is the target branch correct, or should it be 12.1.0?

**Review by marijnritense — APPROVED** [review]

**Pull request merged**